sunpy/coordinates/ephemeris.py

```diff
@@ -61,7 +61,9 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     -----
     There is no correction for aberration due to observer motion.  For a body close to the Sun in
     angular direction relative to the observer, the correction can be negligible because the
-    apparent location of the body will shift in tandem with the Sun.
+    apparent location of the body will shift in tandem with the Sun. As Astropy's built-in ephemeris
+    is not as accurate as JPL ephemeris so you can use `~astropy.coordinates.solar_system_ephemeris`
+    in order to change the ephemeris. You can refer `~astropy.coordinates.solarsystem` for docs.
```

Comment:
This isn't working:
```
/home/runner/work/sunpy/sunpy/.tox/build_docs/lib/python3.9/site-packages/sunpy/coordinates/ephemeris.py:docstring of sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst:12: WARNING: py:obj reference target not found: astropy.coordinates.solarsystem
/home/runner/work/sunpy/sunpy/.tox/build_docs/lib/python3.9/site-packages/sunpy/coordinates/ephemeris.py:docstring of sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst:12: WARNING: py:obj reference target not found: astropy.coordinates.solarsystem
```

Reply:
Can you tell me how to check whether the docs are built or not?

Reply:
If you can read over: https://docs.sunpy.org/en/latest/dev_guide/contents/documentation.html#sphinx  that will walk you through the steps.

Reply:
Thanks